src/main/scala/org/graphframes/GraphFrame.scala

```diff
@@ -442,6 +404,13 @@ class GraphFrame protected(
   def svdPlusPlus(): SVDPlusPlus.Builder = new SVDPlusPlus.Builder(this)
 
   def triangleCount(): TriangleCount.Builder = new TriangleCount.Builder(this)
+
+  /**
+   * This is a primitive for implementing graph algorithms.
+   * This method aggregates values from the neighboring edges and vertices of each vertex.
+   * See [[AggregateMessages]] for detailed documentation.
+   */
+  def aggregateMessages: AggregateMessages.Builder = new AggregateMessages.Builder(this)
```

Comment:
nit: to be consistent with the other algorithms, they should not take empty parens `()`, or this one should.